Update ingestion_mechanisms.md (#15042)
DataDog/documentation: content/en/tracing/trace_pipeline/ingestion_mechanisms.md

For more granular control, use tracing library sampling configuration options:
- Set a specific **sampling rate to apply to all root services** for the library, overriding the Agent's [default mechanism](#in-the-agent).

> - Set a **sampling rate to apply to specific root services** or for specific span operation names.

Author: I think this extra phrase was leaving room for interpretation: I could read this as "or one can also target specific non-root services".

- Set a **rate limit** on the number of ingested traces per second. The default rate limit is 100 traces per second per service instance (when using the Agent [default mechanism](#in-the-agent), the rate limiter is ignored).

> Sampling controls can only be set for root services

Reviewer: ```suggestion
Sampling controls can be set for only root services.
```

> Only root service spans can receive sampling controls.

Reviewer 2: ```suggestion
Sampling controls can only be set for root services
```
